controllers/articleController.js

```diff
@@ -0,0 +1,13 @@
+const { fetchArticles } = require("../models/articleModel");
+
+exports.getArticles = (req, res, next) => {
+    const {article_id} = req.params;
+
+    fetchArticles(article_id)
+    .then((data)=>{
+        res.status(200).send({article: data.rows[0]});
```

Comment:
This could be tidied up by defining what comes back in the model. You already know you want an object with a key of article, you are expecting an article back from the server, with that in mind, the variable in the then block would make sense to be called 'article' that would allow you to send back an object in the short hand ```{article}``` if in the model you defined the first item on the rows array to be returned.